src/pal/src/misc/miscpalapi.cpp

```diff
@@ -454,3 +454,159 @@ EnsureOpenSslInitialized()
     return dwRet;
 }
 
+/*++
```

Comment:
Thanks for the help, guys.  Regarding the name-mangling, so I was able to surround the contents of my .cpp with extern "C", and indeed that removed the name-mangling as visible in the .a file.  What I don't understand, though, is why that name mangling wasn't happening when these functions were in the existing .cpp file, which doesn't wrap the body in extern "C".  Is there some magic happening based on one of the #includes in the other file?  Or something else?